player/brains/superweapon/cPlayerBrainFremenSuperWeapon.cpp

```diff
@@ -32,10 +32,12 @@ namespace brains {
             return;
         }
 
-        char msg[255];
-        sprintf(msg, "cPlayerBrainFremenSuperWeapon::think() AI[%d] - found %d idle unit(s) to attack with.",
-                player->getId(), ids.size());
-        logbook(msg);
+        // char msg[255];
+        // sprintf(msg, "cPlayerBrainFremenSuperWeapon::think() AI[%d] - found %d idle unit(s) to attack with.",
+        //         player->getId(), ids.size());               
+        // logbook(msg);
```

Comment:
No need to leave comments. Just delete code. :) 
We can look in the git history if we need to.